app/src/pages/common/settingsPage/defectTypesTab/defectSubTypeForm.jsx

```diff
@@ -30,23 +35,16 @@ renderColorPicker.propTypes = {
 };
 
 @reduxForm({
-  validate: ({ longName, shortName }) => {
-    const errors = {};
-
-    if (longName === '') {
-      errors.longName = 'requiredFieldHint';
-    } else if (!validate.defectTypeLongName(longName)) {
-      errors.longName = 'defectLongNameHint';
-    }
-
-    if (shortName === '') {
-      errors.shortName = 'requiredFieldHint';
-    } else if (!validate.defectTypeShortName(shortName)) {
-      errors.shortName = 'defectShortNameHint';
-    }
-
-    return errors;
-  },
+  validate: ({ longName, shortName }) => ({
+    longName: composeBindedValidators([
```

Comment:
Please create validators outside of this function to improve readability:

```
longName: composeBindedValidators([
    commonValidators.requiredField,
      bindMessageToValidator(validate.defectTypeLongName, 'defectLongNameHint'),
    ])(longName)
```
vs
```
longName: longNameValidator(longName)
```

Also, check other cases